websockets/handlers/referrer_wsh.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python
+
+from mod_pywebsocket import msgutil
+
+def web_socket_do_extra_handshake(request):
+    pass
+
+def web_socket_transfer_data(request):
+    referrer = request.headers_in.get("Referer")
+    if not referrer:
```

Comment:
I think this should be `if referrer is None`, at least other handlers use that form and I vaguely recall changing to more explicit `None` checks for some reason.